tests/benchmarks/multicore/idle_with_pwm/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	power-states {
+		idle: idle {
+			compatible = "zephyr,power-state";
+			power-state-name = "suspend-to-idle";
+			min-residency-us = <100000>;
+		};
+
+		s2ram: s2ram {
+			compatible = "zephyr,power-state";
+			power-state-name = "suspend-to-ram";
+			min-residency-us = <800000>;
+		};
+	};
+};
```

Comment:
power states should be part of soc dts files.

Reply:
I just checked, power-states are not defined for nrf54h20 at the moment.
I believe that tests in nrf/tests/benchmarks/multicore are more prof-of-concept rather than a final solution.
My task is to confirm that PWM works after leaving low power state.
For now, I prefer to leave it as is.

Reply:
Can you then open a PR adding them to soc dts files?

Reply:
I prefer not. I don't want to be associated with the PM implementation in nrf54h20 :smile:

Reply:
I will regret that
https://github.com/zephyrproject-rtos/zephyr/pull/79098